### DISCUSSION TITLE
drop down select list with scenes for lights

### DISCUSSION THREAD

**Discussion by falkio8** [Clooos/Bubble-Card#1053] [Questions about config, custom styles and templates] [answered]
**I am trying to add a sub button with a drop down list to select scenes for my Hue lights**. Found this for a select card in the official documentation:
```
type: custom:bubble-card
card_type: select
name: Scene
entity: input_select.scenes
icon: mdi:brightness-4
show_state: true
```
I dont quite get it how to use it. Even if I manually create a helper input_select with the names of all relevant scenes by selecting the text in the drop down it doesnt trigger anything on its own. What I am overlooking here? I am aware that the example is not for a sub button but for a card itself but the logic behind it should be the same.

**Comment by Clooos** [comment, marked as answer]
Hi! You need to create an automation for that, basically it's:

if input_select is "Your scene input" then trigger scene.

You need to do that for each values in you input select then it will works 🙂 

There is no easier way for now.

Hope this help!

**Reply by falkio8** [reply]
Ok thank you. Not the answer I wanted to hear :-) 
But I was expecting something like this. Dang

**Reply by Clooos** [reply]
Sorry! That's what I've done for my scenes as well, but once done, it works perfectly well 👌
